src/osd/PrimaryLogPG.cc

```diff
@@ -9620,7 +9626,11 @@ ObjectContextRef PrimaryLogPG::get_object_context(
     dout(10) << __func__ << ": creating obc from disk: " << obc
 	     << dendl;
   }
-  assert(obc->ssc);
+
+  // XXX: Caller doesn't expect this
+  if (obc->ssc == NULL)
+    return ObjectContextRef();   // -ENOENT!
```

Comment:
maybe derr here?  it makes me a bit nervous to turn this into silently an ENOENT when we can't load the snapset.